packages/web/src/hooks/swap/data/use-swap-handler.tsx

```diff
@@ -360,6 +360,14 @@ export const useSwapHandler = () => {
     return "NONE";
   }, [priceImpact]);
 
+  const formatPriceImpact = useCallback((priceImpact: BigNumber | null | undefined) => {
+    if (!priceImpact) return 0;
+
+    if (priceImpact.isLessThan(-100)) return -100;
+
+    return Number(priceImpact.toFixed(2));
+  }, []);
```

Comment:
I think this can reduce the conditions, but it's not necessary to apply this change.

```suggestion
  const formatPriceImpact = useCallback((priceImpact: BigNumber | null | undefined) => {
    if (!priceImpact) return 0;
    return Math.max(-100, Number(priceImpact.toFixed(2)));
  }, []);
```

Reply:
Thanks for the great idea.